apps/native/app/src/screens/home/onboarding-module.tsx

```diff
@@ -1,89 +1,102 @@
 import React from 'react'
+import { useIntl } from 'react-intl'
 import { SafeAreaView, TouchableOpacity } from 'react-native'
 import { useTheme } from 'styled-components/native'
-import { useIntl } from 'react-intl'
 
-import { Close, Heading, ViewPager, WelcomeCard } from '../../ui'
 import illustration1 from '../../assets/illustrations/digital-services-m3.png'
 import illustration3 from '../../assets/illustrations/le-company-s2.png'
 import illustration2 from '../../assets/illustrations/le-retirement-s3-large.png'
 import illustration4 from '../../assets/illustrations/le_jobs_s5.png'
+import { DISMISSED_KEYS } from '../../constants/dissmissed-keys'
+import { useFeatureFlag } from '../../contexts/feature-flag-provider'
 import { useAuthStore } from '../../stores/auth-store'
 import { usePreferencesStore } from '../../stores/preferences-store'
+import { Close, Heading, ViewPager, WelcomeCard } from '../../ui'
+import { HomeBanner } from './home-banner'
 
 export const OnboardingModule = React.memo(() => {
   const theme = useTheme()
   const intl = useIntl()
   const { dismissed, dismiss } = usePreferencesStore()
   const { userInfo } = useAuthStore()
-
-  if (dismissed.includes('onboardingWidget')) {
-    return null
-  }
+  const isHomeBannerEnabled = useFeatureFlag('isHomeBannerEnabled', false)
```

Comment:
~Feature is sometimes called "home banner" and sometimes called "kilometer announcement". Guess we want to refactor this later to support arbitrary banners. 🤷